### ISSUE TITLE
Keyboard navigation and screen reader voicing

### ISSUE THREAD

**Issue by j5v** [j5v/xetune#8]
Currently, the SVG UI is not keyboard navigable, cannot take focus, and therefore provides no hints to screen readers.

- [ ] Research how SVG elements can take keyboard focus
- [ ] Research how SVG elements can have a keyboard-navigable hierarchy
- [ ] Research the aria attributes necessary to generate voice prompts for screen readers.
- [ ] Create voice prompts that give the important information as early as possible, to aid fast navigation.
